src/celengine/globular.cpp

```diff
@@ -214,31 +452,23 @@ float Globular::getConcentration() const
 void Globular::setConcentration(const float conc)
 {
     c = conc;
-    if (!formsInitialized)
-        InitializeForms();
-
     // For saving time, account for the c dependence via 8 bins only,
 
-    form = globularForms[cSlot(conc)];
+    static const std::vector<GlobularForm> globularForms = initializeForms();
```

Comment:
afair, google style guide forbids global/static non-trivial variables, i also remember that we had application failures on exit due to global `std::string`s.

Reply:
Presumably if we're following the Google style guide we should remove all use of `<filesystem>`, as they explicitly forbid that...

Will look at an alternative way to do this.